e2e_tests/test_suites/guestpolicies/guest_policies_utils.go

```diff
@@ -405,7 +405,10 @@ while ($true) {
     `
 		script = fmt.Sprintf("%s\n%s\n%s\n%s", yumInstallInfoPackage, utils.InstallOSConfigEL(image), waitForRestartLinux, scriptLinux)
 	case "zypper":
-		script = fmt.Sprintf("%s\n%s\n%s", utils.InstallOSConfigSUSE(), waitForRestartLinux, scriptLinux)
+		zypperRemoveEdPackageIfExist := `
+    zypper remove -y ed
```

Comment:
If the newlines are really necessary, I'd move them inside the format string in Sprintf.

Reply:
Done, thank you!